### DISCUSSION TITLE
Changing "Hey/Ok Google voice command"

### DISCUSSION THREAD

**Discussion by KaroomiOsu** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#918] [Q&A]
I would like to change the words to something like "Hey PC," is this possible to do?

**Comment by sumeshir26** [comment]
@KaroomiOsu Unfortunately, It is not possible to do this, as the module we use for Hotword Recognition does not easily support custom hotwords :(

**Reply by im-coder-lg** [reply]
You gotta code it in the app and build it.

**Reply by KaroomiOsu** [reply]
Okay, thanks!

**Comment by damianmonelle** [comment]
> You gotta code it in the app and build it.

hey thanks im trying to rebuild it but once i rebuild it and launch it the save button doesnt work and the enter but for text input no longer works any idea what that could be im rebuilding it on mac with conda nodejs
